### ISSUE TITLE
license in block-generic.go

### ISSUE THREAD

**Issue by legrostdg** [minio/md5-simd#16]
I'm packaging md5-simd in Debian. I noticed that the complete LICENSE file of block-generic.go is not included, and that there is also the gen.go source file missing. Could you please add them to the repo?

```
// Copyright 2013 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// Code generated by go run gen.go -output md5block.go; DO NOT EDIT.

**Comment by harshavardhana** [comment]
@legrostdg that is available here https://golang.org/src/crypto/md5/

**Comment by legrostdg** [comment]
OK, thanks. Don't you want to include gen.go and Go's LICENSE file in this repo? That would help packaging... As I understand it, at least the LICENSE file is required by Go's license.

**Comment by harshavardhana** [comment]
> OK, thanks. Don't you want to include gen.go and Go's LICENSE file in this repo? That would help packaging... As I understand it, at least the LICENSE file is required by Go's license.

Go's license is same as our license apache version 2.0

**Comment by legrostdg** [comment]
On 2020-11-17 09:18-0800, Harshavardhana wrote:
>> OK, thanks. Don't you want to include gen.go and Go's LICENSE file in this repo? That would help packaging... As I understand it, at least the LICENSE file is required by Go's license.
>
>Go's license is same as our license apache version 2.0

It does not seem to be the case. Go's license seems to be MIT/Expat
https://golang.org/LICENSE?m=text

-- 
Félix

**Comment by harshavardhana** [comment]
Closed in b1172e05418016babe82eafdcdb6508cab98a87b
